dropbox/dropbox.py

```diff
@@ -211,11 +218,25 @@ def request(self,
         route_style = route.attrs['style'] or 'rpc'
         serialized_arg = stone_serializers.json_encode(route.arg_type,
                                                        request_arg)
+
+
+        if (timeout is None and
+            route == files.list_folder_longpoll):
```

Comment:
One more indent on this line continuation. Also, good way to do it (specifically checking the route)!